action/login/prompt.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package login
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/manifoldco/promptui"
+
+	"github.com/sirupsen/logrus"
+)
+
+// PromptUsername asks the user to provide a username via
+// terminal input based off the provided configuration.
+func (c *Config) PromptUsername(in io.ReadCloser) error {
```

Comment:
This is where we add the new `PromptUsername()` function.

The purpose of this function is to create a prompt for the username via the terminal session.